client/src/apps/code-collabo/styles/app.main.js

```diff
@@ -22,6 +22,24 @@ export default css.global`
 
     }
 
+    .app__logo {
+        margin-bottom: 40px;
+    }
+
+    .app__menubar__nav__link {
+        text-decoration: none;
+        background: #f9f9f9;
+        padding: 15px 0px;
+        gap: 24px;
+        border-radius: 10px;
+    }
+
+    .app__menubar__nav__items {
```

Comment:
Both Mobile and Desktop:
Let `.app__menubar__nav__items` styles class come before `.app__menubar__nav__link` since that is the container.

Reply:
@Ifycode Done